Analysis/analysis.tex

```diff
@@ -57,17 +59,18 @@ \subsection{Pileup Reweighting}
   }
   \label{fig:pudist}
 \end{figure}
+\PH{This setion is completely nebulous, you don't actually expalin that you are deriving a weight. This needs a rewrite}
 
 \section{Event Selection}
 \label{sec:event_selection}
 
-From the recorded data, events are selected by requiring $\met > 170\GeV$ and at least one photon with $\ETg > 175\GeV$ in the fiducial region of the ECAL barrel ($\abs{\eta} < 1.44$). Events with photons in the endcaps are not considered because the estimate of backgrounds due to beam halo and misidentified hadron are greatly complicated due to the $x$-$y$ grid of the crystals in the endcaps.
+From the recorded data, events are selected by requiring $\met > 170\GeV$ and at least one photon with $\ETg > 175\GeV$ in the fiducial region of the ECAL barrel ($\abs{\eta} < 1.44$). Events with photons in the endcaps are not considered because the estimate of backgrounds due to beam halo and misidentified hadron are greatly complicated due to the $x$-$y$ grid of the crystals in the endcaps. \PH{This is very hard to follow can you put $x$-$y$ grid in layman's terms}
```

Comment:
rephrased with the more important point that the endcap does not have an $eta$ $\phi$ grid